src/libstd/macros.rs

```diff
@@ -64,6 +64,9 @@ macro_rules! panic {
 ///
 /// Equivalent to the `println!` macro except that a newline is not printed at
 /// the end of the message.
+///
+/// Note that stdout is frequently line-buffered by default so it may be necessary to use
+/// io::stdout().flush() to ensure the output is emitted immediately.
```

Comment:
Can you match the surrounding style and wrap this to 80-characters as well? Also, can you put `io::stdout().flush()` in backticks as well?